app/assets/stylesheets/content.scss

```diff
@@ -33,9 +54,9 @@
 }
 
 #upvoted {
-  color: #BDD860;
+  color: #bdd860;
```

Comment:
Color literals like `#bdd860` should only be used in variable declarations; they should be referred to via variable everywhere else.